lib/galaxy/tools/data_manager/manager.py

```diff
@@ -57,7 +57,7 @@ def load_from_xml( self, xml_filename, store_tool_path=True, replace_existing=Fa
                 tool_path = '.'
             self.tool_path = tool_path
         for data_manager_elem in root.findall( 'data_manager' ):
-            self.load_manager_from_elem( data_manager_elem, replace_existing=replace_existing )
+            self.load_manager_from_elem( data_manager_elem, tool_path=tool_path, replace_existing=replace_existing )
```

Comment:
This should be `tool_path=self.tool_path` because `tool_path` is undefined if `store_tool_path` is False.